src/models/ComponentLink.js

```diff
@@ -53,6 +56,11 @@ class ComponentLink extends FileInformation {
      * @type {ComponentLinkDefinition}
      */
     this.definition = definition || null;
+    /**
+     * Indicate if this link is a reverse link.
```

Comment:
The pb of this kind of documentation, is that it's not clear at all. What is a reverse link ?